autotst/reaction.py

```diff
@@ -685,20 +685,15 @@ def ase_molecule(self):
             self._ase_molecule = self.get_ase_mol()
         return self._ase_molecule
 
-    def get_rdkit_mol(self):
+    def get_pseudo_geometry(self):
         """
-        A method to create an rdkit geometry... slightly different than that of the conformer method
-        returns both the rdkit_molecule and the bm
+        A method to create a _pseduo_geometry. Essentailly an RDKit molecule with a fake bond drawn
+        between reacting atoms which are not bound to be used in other processes 
         """
-        self._rdkit_molecule = Conformer(rmg_molecule=self.rmg_molecule).get_rdkit_mol()
-
-        self.get_labels()
-        for i, atom in enumerate(self.rmg_molecule.atoms):
-            assert atom.number == self.rdkit_molecule.GetAtoms()[i].GetAtomicNum()
-
+        
         if len(self.labels) == 3:
```

Comment:
We could add the `self.get_labels` here as a precaution. And the  `_pseudo_geometry` is used for the identification of dihedrals, angles, and bonds for TSs because it draws a fake bond between reacting atoms if one doesn't exist. And I _believe_ that @skrsna uses this 🤔 . I'll add the `self.get_labels` here though.

Reply:
I was using `ts.rdkit_molecule` which I guess has 3D coords. I'll see if `_pseudo_geometry` is more efficient